nix/mobile/android/release.nix

```diff
@@ -46,7 +46,7 @@ in stdenv.mkDerivation rec {
       root = path;
       include = [
         "package.json" "yarn.lock" "metro.config.js" "babel.config.js"
-        "resources/.*" "translations/.*" "src/js/worklet_factory.js"
+        "resources/.*" "translations/.*" "src/js/worklet_factory.js" "src/js/bottom_sheet.js"
```

Comment:
This is already covered by `"src/js/.*"` used in #14410 by Parvesh. So you might as well do the same thing.